runtime/src/main/java/com/dylibso/chicory/runtime/Module.java

```diff
@@ -268,12 +267,16 @@ private HostImports mapHostImports(Import[] imports, HostImports hostImports) {
             switch (imprt.getDesc().getType()) {
                 case FuncIdx:
                     hostFuncNum++;
+                    break;
```

Comment:
oops 😆

Reply:
That precise feeling 😅

Reply:
For me this is the downside of writing in so many languages. I forget all these tiny little differences. I wish Java did more to protect you here. I like the enhanced switch of course, but we can't use that yet right?. Maybe some tooling could help?

Reply:
> For me this is the downside of writing in so many languages. 

same here, I think that I wrote this bug by myself even if I'm using Java daily.

> we can't use that yet right?

we would need to switch to Java 17, which can be a reasonable choice as even [Quarkus is dropping Java 11 support](https://quarkus.io/blog/java-17/), I have mixed feelings about ...